dali/core/geom_vec_test.cu

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "dali/test/device_test.h"
+#include "dali/core/geom/vec.h"
+
+namespace dali {
+template <size_t N, typename T>
+__device__ DeviceString dev_to_string(const vec<N, T> &v) {
```

Comment:
```suggestion
__device__ DeviceString dev_vec_to_string(const vec<N, T> &v) {
```

Reply:
The name matters - it's used by DEV_EXPECT_EQ to print the value.